docs/source/dev_guide/algorithms/pca/computation-batch.rst

```diff
@@ -0,0 +1,175 @@
+.. ******************************************************************************
+.. * Copyright 2020 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+Batch Processing
+****************
+
+.. contents::
+    :local:
+    :depth: 1
+
+Algorithm Input
+---------------
+The PCA algorithm accepts the input described below. Pass the
+Input ID as a parameter to the methods that provide input for your algorithm.
```

Comment:
"For more details, see Algorithms." is missing here.

Reply:
fixed